ion/src/simulator/shared/main_sdl.cpp

```diff
@@ -101,19 +135,27 @@ void relayout() {
   SDL_GetWindowSize(sWindow, &windowWidth, &windowHeight);
   SDL_RenderSetLogicalSize(sRenderer, windowWidth, windowHeight);
 
-#if EPSILON_SDL_SCREEN_ONLY
-  sScreenRect.x = 0;
-  sScreenRect.y = 0;
-  sScreenRect.w = windowWidth;
-  sScreenRect.h = windowHeight;
-#else
-  Layout::recompute(windowWidth, windowHeight);
-  SDL_Rect backgroundRect;
-  Layout::getBackgroundRect(&backgroundRect);
-
-  SDL_RenderCopy(sRenderer, sBackgroundTexture, nullptr, &backgroundRect);
-  SDL_RenderPresent(sRenderer);
-#endif
+  if (argument_screen_only) {
+    // Keep original aspect ration in screen_only mode.
+    float scale = (float)(Ion::Display::Width) / (float)(Ion::Display::Height);
+    if ((float)(windowHeight) * scale > float(windowWidth)) {
```

Comment:
Is this `(float)` casts necessary, or they're here just for readability, because scale is anyway float, thus `int` argument will be anyway implicitly converted to `float`. Same for two lines below.